README.md

```diff
@@ -98,28 +100,28 @@ let number = CardNumber(rawValue: "5555 5555 5555 4444")
 //Now create a card object. You can optionally validate a card.
 let card = Card(name: name, number: number, cvv: cvv, expiry: expiry)
 
-// Create a card create operation
+// Create a createCardOperation
```

Comment:
`createCardOperaton` is just an example property name, class name is `CreateCard`. I'd suggest:
> Create a `CreateCard` operation

Reply:
Or actually, since it's a code snippet, just:
> Create a create card operation

?

Reply:
Makes sense, but it's still tough to read. Maybe something instead of the first "create"?

- Initialise a create card operation
- Perform...

Reply:
Initialize is good 👍 
> Initialize a create card operation